tests/unit/accounts/test_token_revocation.py

```
@@ -0,0 +1,159 @@
+from accounts.models import JwtToken
+from accounts.token_revocation import revoke_all_user_tokens
+
+
+class BaseAPITestClass(APITestCase):
+    def setUp(self):
+        self.client = APIClient(enforce_csrf_checks=True)
+
+        self.user = User.objects.create(
+            username="someuser",
+            email="user@test.com",
+            password="secret_password",
```

Ruff (0.15.18)

[error] Possible hardcoded password assigned to argument: "password"

(S106)